### DISCUSSION TITLE
Does any one have problems replicating the following example with Dashboards?

### DISCUSSION THREAD

**Discussion by riosavila** [quarto-dev/quarto-cli#8605] [Q&A] [answered]
### Description

So, im trying the new Dashboard feature using the example code below

````
  --- 
  title: "Development Indicators by Continent"
  author: "Gapminder Analytics Group"
  format: dashboard
  --- 
  
  ```{python}
  import plotly.express as px
  df = px.data.gapminder()
  ```
  
  ## Row {height=60%}
  
  ```{python}
  #| title: GDP and Life Expectancy 
  px.scatter(  
    df, x="gdpPercap", y="lifeExp", 
    animation_frame="year", animation_group="country", 
    size="pop", color="continent", hover_name="country",
    facet_col="continent", log_x=True, size_max=45, 
    range_x=[100,100000], range_y=[25,90] 
  )  
  ```
  
  ## Row {height=40%}
  
  ```{python}
  #| title: Population
  px.area(
    df, x="year", y="pop", 
    color="continent", line_group="country"
  )
  ```
  
  ```{python}
  #| title: Life Expectancy
  px.line(
    df, x="year", y="lifeExp", 
    color="continent", line_group="country"
  )
  ```

````

However, while it produces the correct outputs as html, whenever using dashboards, it produces blank pages.

**Comment by cscheid** [comment]
I get the following output with your code:

<img width="1483" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/285675/004dbe8b-7b05-4bda-8c02-d1fd4953dfcc">

**Reply by riosavila** [reply]
Thank you.
that is what i wonder.
Everytime I run it i cant get this outcome.
Something else to add, when rendering this with vsc it gives me 

 /plotly.js (404: Not Found)

But not sure how to fix that

**Reply by cscheid** [reply]
Are you trying to double-click on the rendered file instead of using the preview server or a local web server?

**Comment by cderv** [comment, marked as answer]
@riosavila are you on windows ? 

I missed this two week ago (sorry), but I believe you are encountering this issue: 
* https://github.com/quarto-dev/quarto-cli/issues/8857

We'll fix soon.

**Reply by cderv** [reply]
This will be fixed in next pre-release 1.5, but also backported in our next 1.4 patch release.
